browser/ui/browser_commands.h

```diff
@@ -22,7 +23,8 @@ void ShowBraveVPNBubble(Browser* browser);
 void ToggleBraveVPNButton(Browser* browser);
 void OpenBraveVPNUrls(Browser* browser, int command_id);
 void OpenIpfsFilesWebUI(Browser* browser);
-void CopyCleanLink(Browser* browser);
+void CopyCleanLink(Browser* browser, const GURL& url);
+void CopyLinkWithStrictCleaning(Browser* browser, const GURL& url);
```

Comment:
would be great to have comments for these functions, b/c it is not really clear what is clean and what is strictly clean